.github/workflows/apt-release.yml

```diff
@@ -0,0 +1,61 @@
+name: Apt Release
+on:
+  workflow_dispatch:
+  workflow_run:
+    # wait for Cli Release to complete, when it completes, start this workflow
+    workflows: ["Cli Release"]
+    branches: [main]
+    types:
+      - completed
+
+jobs:
+  apt-build-release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check Out Repo
+      uses: actions/checkout@v2
+    - name: Fetch tag
+      id: get-tag
+      run: |
+        git fetch --tags
+        echo "::set-output name=TAG_NAME::$(git describe --tags $(git rev-list --tags --max-count=1))"
+    - name: Configure AWS credentials from Test account
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }}
+        aws-region: us-east-1
+
+    - name: generate linux tarball
+      run: |
+        npm cache clear & npm install
+        npm install -g @oclif/dev-cli
+        oclif-dev pack --targets=linux-x64
+        
+    - name: Build Deb Package
+      env:
+        GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY}}
+        GPG_SIGNING_KEY_ID: ${{ secrets.GPG_SIGNING_KEY_ID }}
+        GPG_SIGNING_KEY_PASSPHRASE: ${{ secrets.GPG_SIGNING_KEY_PASSPHRASE }}
+      run: | 
+        node .github/scripts/pack-debian x64
+
+  notify-complete-fail:
+    if: ${{ failure() || cancelled() }}
```

Comment:
From Ravali's other PR, she has [modified](https://github.com/twilio/twilio-cli-core/blob/fe4fcd5238973a4e276318281113002722955759/.github/workflows/cli-core-test.yml#L34) the criteria for notification, would that change be required here too?

Reply:
So this workflow is triggered during the release [process](https://github.com/twilio/twilio-cli/pull/320/files#diff-eea288d670221a2e5dacb262e06da62c428e407712470af2d5aede898cfd4853R6), whereas the the CLI and CLI Core test is triggered whenever there is a push to main branch or a pull request is created see [here](https://github.com/twilio/twilio-cli-core/blob/main/.github/workflows/cli-core-test.yml#L3-L5). The changes made by Ravali's PR is to prevent failure notification triggered during a pull request.

Reply:
Got it! 👍